File: SpringWeb/SpringWebTest-1/src/main/java/com/example/demo/config/JPAConfiguration.java
package com.example.demo.config;

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EntityScan(basePackages = {"com.example.demo"})
@EnableJpaRepositories(basePackages = "com.ohgiraffers.datajpa")
public class JPAConfiguration {
	
}

File: Spring_OAuth2/securityserver/src/main/kotlin/com/example/securityserver/configure/JPAConfiguration.kt
package com.example.securityserver.configure

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EntityScan(basePackages = ["com.example.securityserver"])
@EnableJpaRepositories(basePackages = ["com.example.securityserver"])
public class JPAConfiguration {

}
